conans/test/integration/command/info/info_test.py

```diff
@@ -356,6 +354,26 @@ def export(self):
         assert "ERROR: Package 'dep/0.1' not resolved: dep/0.1: Cannot load" in c.out
         assert exit_code == ERROR_GENERAL
 
+    def test_invalid_config_validate(self):
```

Comment:
Instead of a new test, I think a new check in the existing ``test_validate_create`` test would be easier (and faster)

Reply:
I knew there had to be a test for it already, but could not find it, thanks for the tip :)